### PR TITLE
6369 remove SVM tests from ZFS test suite

### PR DESCRIPTION
**Pull request by ahrens**
https://www.illumos.org/issues/6369

At this point, I don't think anyone cares about how ZFS interacts with SVM; let's remove the tests (which don't work anyway on some illumos distros).

### CODE REVIEW

**Comment by ahrens** [comment]
@zettabot go

**Comment by ahrens** [comment]
regression tests completed: http://build.prakashsurya.com:8080/job/openzfs-regression-tests/36/

File: usr/src/test/zfs-tests/include/libtest.shlib [hunk 1]
```diff
@@ -1695,7 +1678,6 @@ function find_disks
 	max_finddisksnum=${MAX_FINDDISKSNUM:-6}
 
 	$SWAP -l > $sfi
-	$METASTAT -c > $msi 2>/dev/null
```
Comment (jwk404) [inline usr/src/test/zfs-tests/include/libtest.shlib]: This function has other references to $msi that should be deleted as well.

Comment (ahrens) [inline usr/src/test/zfs-tests/include/libtest.shlib]: Thanks for pointing that out.  I've fixed it now.

**Comment by trisk** [comment]
LGTM.

**Comment by ahrens** [comment]
closed by:

commit 2bc93fe62512a6abb89bba61b74bc14c683306ad
Author: Matthew Ahrens <mahrens@delphix.com>
Date:   Wed Oct 28 09:52:33 2015 -0700

    6369 remove SVM tests from ZFS test suite
    Reviewed by: John Kennedy <john.kennedy@delphix.com>
    Reviewed by: Prakash Surya <prakash.surya@delphix.com>
    Reviewed by: Albert Lee <trisk@omniti.com>
    Approved by: Dan McDonald <danmcd@omniti.com>

**Pull request closed without merging**